Add user agent filter to public documentation (#408)
ngrok/ngrok-docs: docs/secure-tunnels/ngrok-agent/reference/config.md

| `schemes`                    | `http`, `https`                                                              | bind to an HTTPS endpoint (                                                                                                                                                                                                                                                                       |
| `subdomain`                  | Any valid combination of letters, numbers, hyphens or periods.               | subdomain name to request. If unspecified, ngrok provides a unique subdomain based on your account type.                                                                                                                                                                                          |

> | `ua_filter_allow`            | Array of Strings                                                             | Allows User-Agents that match against these regexes                                                                                                                                                                                                                                               |

Reviewer: This doesn't work for me, giving me `ERROR:    line 8: field ua_filter_allow not found in type config.HTTPv2Tunnel`.

What does work is:

```yaml
# test.yml
version: 2

tunnels:
  test:
    proto: "http"
    addr: "127.0.0.1:8080"
    user_agent_filter:
      allow: ["/foo/"]
      deny: ["/bar/"]
```

```command
$ ngrok start --config test.yml --all
```

I think the fully written out form there is intended, so I think this is meant to be `user_agent_filter.allow` and then `user_agent_filter.deny` below.

Small nit, but let's write out "regexes" as "regular expressions", possibly even as "RE2 Regular Expressions", since otherwise people might assume it's PCRE.

> | `ua_filter_allow`            | Array of Strings                                                             | Allow User-Agents that can be matched against these regex                                                                                                                                                                                                                                         |
> | `ua_filter_deny`             | Array of Strings                                                             | Deny User-Agents that can be matched against these regex                                                                                                                                                                                                                                          |

Reviewer 2: another nit, the tense reads a bit weird here for me:
```suggestion
| `ua_filter_allow`            | Array of Strings                                                             | Allows User-Agents that match against these regexes                                                                                                                                                                                                                                         |
| `ua_filter_deny`             | Array of Strings                                                             | Denies User-Agents that match against these regexes                                                                                                                                                                                                                                          |
```
